store/README.md

```diff
@@ -18,6 +18,7 @@ mkdir -p ../builds/default/v3/
 curl https://app.yunohost.org/default/v3/apps.json > ../builds/default/v3/apps.json
 
 # You will also want to run list_builder.py to initialize the .apps_cache (at least for a few apps, you can Ctrl+C after a while)
+pip3 install tqdm GitPython
```

Comment:
Wouldn't it make more sens in a requirements.txt? Also with a major version pinned to be future proof.

Reply:
Note that the `list_builder.py` is not part of the store app